metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-dhcp",
-  "version": "5.0.2-rc0",
+  "version": "5.1.0-rc0",
```

Comment:
Version gets changed only when we do a release PR.

```suggestion
  "version": "5.0.2-rc0",
```

Reply:
Just remove the last commit and it should be good.  From your branch:
```
git reset --hard HEAD^
git push -f
```

Reply:
I've reverted that change.